client/app/lib/helpers/url-helpers.js

```diff
@@ -21,5 +21,25 @@ function getUrlParameter(sParam) {
   return '';
 }
 
+/**
+ * Get the course id from URL.
+ *
+ * return {number}
+ */
+function getCourseId() {
+  const match = window.location.pathname.match(/^\/courses\/(\d+)/);
+  return match && match[1];
+}
+
+/**
+ * Get the survey id from URL.
+ *
+ * return {number}
+ */
+function getSurveyId() {
```

Comment:
More like `define_getter`, where we can declare getSurveyId in the following manner:
 `getSurveyId = define_getter(/^\/courses\/\d+\/surveys\/(\d+)/)`